ppg-server/control

```diff
@@ -0,0 +1,21 @@
+Source: percona-ppg-server-14
+Section: database
+Priority: optional
+Maintainer: Percona Development Team <info@percona.com>
+Standards-Version: 14.4 
+Homepage: www.percona.com
```

Comment:
Should this be the same URL as noted in the RPM spec file? Also it should probably be prepended with `https://`...